### ISSUE TITLE
Crash for init CachingPlayerItem

### ISSUE THREAD

**Issue by andy90s** [sukov/CachingPlayerItem#7]
when add a looper on queueplayer,  crash for init

like this:
```swift
looper = AVPlayerLooper(player: queuePlayer, templateItem: loopPlayerItems.first!)
```
how can I  resolve this?

**Comment by sukov** [comment]
@andy90s Can you please provide the entire code block along with CachingPlayerItem init?

**Comment by sukov** [comment]
The crash on init is fixed in the latest release [1.0.7](https://github.com/sukov/CachingPlayerItem/releases/tag/1.0.7)
